src/main/java/com/ivoyant/kafkaemailevent/service/EmailService.java

```diff
@@ -46,4 +53,23 @@ public String createEmailEvent(EmailDto emailDto) {
         return email.getEmailId();
     }
 
+    public String createEmailAttachmentEvent(EmailAttachDto emailAttachDto) {
+        EmailAttach emailAttach = EmailAttach.builder().emailId(UUID.randomUUID().toString())
+                .fromEmail(emailAttachDto.getFromEmail()).toEmail(emailAttachDto.getToEmail())
+                .subject(emailAttachDto.getSubject()).body(emailAttachDto.getBody()).createdAt(Instant.now())
+                .attachment(emailAttachDto.getAttachment()).build();
+        LOGGER.info("email with attachment created and saved to cassandra cluster {}", emailAttach.getEmailId());
+        emailAttachRepository.save(emailAttach);
+        CompletableFuture<SendResult<String, Object>> future = template.send("email-attach-event", emailAttachDto);
+        future.whenComplete((result, ex) -> {
+            if (ex == null) {
+                LOGGER.info("Sent message=[" + emailAttachDto.toString() +
```

Comment:
Use placeholders in loggers instead of String concatenation. 

Example: 
`LOGGER.info("send message and message is {}", messageString);`